rcl_yaml_param_parser/src/parser.c

```diff
@@ -826,6 +1136,77 @@ void rcl_yaml_node_struct_print(
       }
     }
   }
+  printf("\n Node Name\t\t\t\tParameter Descriptors\n");
```

Comment:
@bpwilcox nit: I guess it's fine to begin with, as it's precludes the need to match parameter values with parameter descriptors, but it'd be nice to collapse both printed tables into one.

Reply:
Right, I didn't want to have to match values and descriptors.